vivisect/parsers/elf.py

```diff
@@ -246,14 +293,26 @@ def loadElfIntoWorkspace(vw, elf, filename=None):
                     elif rtype == Elf.R_386_32:
                         pass
 
+                    elif rtype == Elf.R_X86_64_GLOB_DAT:
+                        vw.makeImport(rlva, "*", name)
+                        vw.setComment(rlva, dmglname)
+
                     else:
                         vw.verbprint('unknown reloc type: %d %s (at %s)' % (rtype, name, hex(rlva)))
+                        
 
             if arch == 'arm':
-                name = r.getName()
                 if name:
                     if rtype == Elf.R_ARM_JUMP_SLOT:
+                        vw.makeImport(rlva, "*", dmglname)
+
+                    elif rtype == Elf.R_ARM_ABS32:      # Direct 32 bit  */
+                        vw.makeImport(rlva, "*", name)
+                        vw.setComment(rlva, dmglname)
+
+                    elif rtype == Elf.R_ARM_GLOB_DAT:   # Create GOT entry */
                         vw.makeImport(rlva, "*", name)
+                        vw.setComment(rlva, dmglname)
```

Comment:
condense all these conditionals into this maybe (since all the sub blocks look the same)?
`if rtype in (Elf.R_ARM_JUMP_SLOT, Elf.R_ARM_ABS32, Elf.R_ARM_GLOB_DAT):`

Reply:
these get heavily overhauled in my current delta.  not sure whether to wrap them into this PR, or wait and have them a part of the atlas_arm_fixes PR